package.json

```diff
@@ -8,6 +9,10 @@
     "packages/stream"
   ],
   "scripts": {
+    "changeset": "changeset",
+    "release": "changeset publish",
+    "version-bump": "changeset version",
+    "postinstall": "manypkg fix",
```

Comment:
`manypkg` is a nice little helper that just checks all of your packages to check if you have any common monorepo mistakes (ie version incompatibilities). It's pretty sweet and simple and `fix` just neatly autofixes anything that it can.